.circleci/helm-values/staging.proxy-scram.yaml

```diff
@@ -7,6 +7,7 @@ image:
 settings:
   authBackend: "console"
   authEndpoint: "http://console-staging.local:9095/management/api/v2"
```

Comment:
by the way, this is wrong. AWS ALB do not serve 9095 port, it serve 80 (and pass traffic to port 9095 in target pods) only for internal console hostnames.

It should be simple `authEndpoint: "http://console-staging.local/management/api/v2"`